atomic_reactor/plugins/exit_store_metadata.py

```diff
@@ -61,46 +61,40 @@ def get_digests(self):
         """
 
         digests = {}  # repository -> digest
-        for registry in self.workflow.push_conf.docker_registries:
+        for _, registry_conf in self.workflow.conf.registries.items():
```

Comment:
this might still give the impression that we should iterate through registries, but this will run only once, right?

Reply:
that brings me to another thought. We decided to keep the "registries" part in the reactor config map to allow backward compatibility, but in code we don't need to reference "registries" and we can change it, or no? If we could change it to single "registry" we would not even need to keep using dict with just 1 key.

Reply:
yes because we're only allowing config for one registry

Reply:
hmm, I agree as well. I have updated it to be a config dict only instead of uri to dict mapping.